<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>KoPageLayoutHeaderBase</class>
 <widget class="QWidget" name="KoPageLayoutHeaderBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>344</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="grpFootEndnote" >
     <property name="title" >
      <string>Footnote/ Endnote</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblFootEndnoteDesc" >
        <property name="text" >
         <string>Spacing between footnote and body:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="grpFooter" >
     <property name="title" >
      <string>Footer</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" colspan="2" >
       <widget class="QCheckBox" name="rfEvenOdd" >
        <property name="text" >
         <string>Different footer for even and odd pages</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="rfFirst" >
        <property name="text" >
         <string>Different footer for the first page</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lblFooterDesc" >
        <property name="text" >
         <string>Spacing between footer and body:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="grpHeader" >
     <property name="title" >
      <string>Header</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <widget class="QLabel" name="lblHeaderDesc" >
        <property name="text" >
         <string>Spacing between header and body:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QCheckBox" name="rhEvenOdd" >
        <property name="text" >
         <string>Different header for even and odd pages</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="rhFirst" >
        <property name="text" >
         <string>Different header for the first page</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
